src/odemis/gui/cont/acquisition/sparc_acq.py

```diff
@@ -65,6 +65,13 @@ def __init__(self, tab_data, tab_panel, streambar_controller):
         self._main_data_model = tab_data.main
         self._tab_panel = tab_panel
         self._streambar_controller = streambar_controller
+        self._acquisition_info = {}  # info priority value (int): (info lvl (int), info message (str))
+        self._interlockTrigger = False  # local/private bool to track interlock status
```

Comment:
Please name it `self._interlockTriggered`, to match the name of the VA, it's simpler.